cpu/nrf52/periph/i2c.c

```diff
@@ -16,13 +17,16 @@
  *
  * @author      Dimitri Nahm <dimitri.nahm@haw-hamburg.de>
  * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
  *
  * @}
  */
 
 #include <string.h>
+#include <errno.h>
 
 #include "cpu.h"
+#include "periph_conf.h"
```

Comment:
I don't think this include is required

Reply:
It is, for initialize pins and speed.

Reply:
Oh I removed this include and this still compiles...
Looks like I can remove it then.

Reply:
That's weird btw. Maybe this include should be added to `i2c.h`, this is also the case for spi, uart, etc

Reply:
Probably, but I think we can leave it as it right now.

Reply:
The periph_conf.h is usually include indirectly via cpu.h and periph_cpu.h